doc/releases/quincy.rst

```diff
@@ -5,6 +5,151 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.4 Quincy
+==============
+
+This is the 4th  backport release in the Quincy series. We recommend
+all users update to this release.
+
+Notable Changes
+---------------
+
+* Cephfs: The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard
+  'AT_STATX_DONT_SYNC' macro. The 'AT_NO_ATTR_SYNC' macro will be removed in
+  the future.
```

Comment:
This is only for coders?  If so maybe mention that so users don't scratch their collective heads.

Reply:
+1.

Still scratching.

Reply:
@batrick @vshankar ^